### DISCUSSION TITLE
How does a code handle such problems when both initial and boundary conditions are variables

### DISCUSSION THREAD

**Discussion by Scienkiller** [lululxvi/deepxde#834] [Q&A]
Hello!
    Thanks to the author for his work! It's very interesting and meaningful. I have a problem that has been bothering me for a long time. As far as I understand, DeepONET can handle different boundary conditions under the same initial conditions, or the same boundary problem under different initial conditions. But how does DeepONET handle if both initial and boundary conditions are variables.
    For example, We want to obtain a steady-state temperature field, then when the boundary conditions are given, the final result should be the same no matter what initial temperature distribution.

**Comment by lululxvi** [comment]
> For example, We want to obtain a steady-state temperature field, then when the boundary conditions are given, the final result should be the same no matter what initial temperature distribution.

Then it only depends on BC and you can use DeepONet.
